inc/impact.class.php

```diff
@@ -895,6 +895,22 @@ public static function searchAsset(
          );
       }
 
+      if ($item->mayBeDeleted()) {
+         $criteria["$table.is_deleted"] = 0;
+         $base_request = array_merge(
+            $base_request,
+            $criteria
+         );
+      }
+
+      if ($item->mayBeTemplate()) {
+         $criteria["$table.is_template"] = 0;
+         $base_request = array_merge(
+            $base_request,
+            $criteria
+         );
```

Comment:
```suggestion
         $base_request['WHERE']["$table.is_template"] = 0;
```

Reply:
At the beginning, i do this, but without array_merge, the SQL do not use parameters
`SELECT `glpi_computers`.`id`, `glpi_computers`.`name` FROM `glpi_computers` WHERE (`glpi_computers`.`entities_id` IN ('1')) LIMIT 20`